views/pages/create.ejs

```diff
@@ -109,6 +109,7 @@
     <script language="javascript">
         function setElementsState(state) {
             ['cols', 'rows', 'carrier', 'battleship', 'destroyer', 'patrol-boat'].forEach((id) => {
+                document.getElementById(id).value = Math.floor(document.getElementById(id).value)
```

Comment:
Fair point, while I'm sure that `Math.round` and `Math.floor` both have similar amounts of efficiency, I had assumed that `Math.floor` would be slightly more efficient as it only has to truncate the value, but `Math.trunc` may be far more efficient in that case. Since the form already doesn't accept values between two numbers, there would be no reason to try and round to the nearest number as (per my testing) you may only submit values such as 1.0, 5.0, 10.0, etc.
I'll change it to Math.trunc in the meantime.